tests/src/test/java/alluxio/client/rest/TestCase.java

```diff
@@ -164,6 +164,16 @@ public HttpURLConnection execute() throws Exception {
     }
 
     connection.connect();
+    return connection;
+  }
+
+  /**
+   * if response is not successful, it will assert fail.
+   * @return connection
+   * @throws Exception
+   */
+  public HttpURLConnection verifiedExecute() throws Exception {
```

Comment:
```suggestion
  public HttpURLConnection executeAndAssertSuccess() throws Exception {
```

You'll need to refactor the name elsewhere too.

Reply:
done